v4-client-py/v4_client_py/clients/helpers/chain_helpers.py

```diff
@@ -20,15 +20,17 @@ class OrderTimeInForce(Flag):
     FOK = auto()    # Fill or Kill
 
 class OrderExecution(Flag):
-    DEFAULT = auto()    # Default, STOP_LIMIT and TAKE_PROFIT_LIMIT
-    POST_ONLY = auto()  # Post-only, STOP_LIMIT and TAKE_PROFIT_LIMIT
+    DEFAULT = auto()    # Default
     IOC = auto()        # Immediate or Cancel
+    POST_ONLY = auto()  # Post-only
     FOK = auto()        # Fill or Kill
```

Comment:
This was not matching the order defined in the [`TimeInForce` proto](https://github.com/dydxprotocol/v4-chain/blob/c41ce4cd261121a41a5b148238349ef5b5490d45/proto/dydxprotocol/clob/order.proto#L157-L178) which I believe is a bug.

Reply:
Synced in-person, ordering matters for this since it's used for initializing the `Order.TimeInForce` field.